### ISSUE TITLE
When called in an azure function callback does not invoke

### ISSUE THREAD

**Issue by kobyl** [Azure/azure-storage-node#573]
### Which service(blob, file, queue, table) does this issue concern?
Table

### Which version of the SDK was used?
2

### What's the Node.js/Browser version?
Latest azure function host. 

### What problem was encountered?
From an async azure function I can invoke insertOrReplaceEntity(), but my callbackfunction is never called back. The approximate code is:

```
await new Promise( (resolve, reject) => {
  tableService.insertOrReplaceEntity(..., (error) => {
      context.log("Insert complete!");
      if(error) reject(error):
      else resolve();
  });
});
```

This same code works in the browser when deployed on my local version of node: 10.14.1, but it hangs when I deploy it to azure. I do see the entity inserted but I never see the "Insert Complete" notification.

### Steps to reproduce the issue?
Create an async azure javascript function. Insert an entity into a tablestore account. Observe that it doesn't return.

### Have you found a mitigation/solution?
No. :(

**Comment by XiaoningLiu** [comment]
@kobyl Can you provide a sample stably re-produce the issue? Does this issue always happen in Azure Function?

**Comment by kobyl** [comment]
I figured it out - when I was doing testing I was running my code in the browser, so had

```
  let azure = require('azure-storage/...browser/...');
```

When I deployed to node I forgot that I was importing the browser version. Once I switched it to:

```
  let azure = require('azure-storage');
```

...my callbacks came back properly.

@XiaoningLiu - thanks for replying! It was in preparing the sample for you that I noticed my error. I do think that this should be called out as a 'gotcha' though. when I was developing I remember I ran into the reverse problem - importing azure-storage and having it break on the browser.
